e2e-tests/experiments/functional/lvm-controller-high-availability/README.md

```diff
@@ -0,0 +1,38 @@
+## About this experiment
+
+This functional experiment scale up the lvm-controller replica to use it in high availability and then verifies the lvm-localpv behaviour when one of the replica goes down. This experiment checks the starting no of replicas of lvm-controller and scale it by one if a free node is present which should be able to schedule the pods.
+
+## Supported platforms:
+
+K8S : 1.17+
+
+OS : Ubuntu
+
+LVM : 2
+
+## Entry-Criteria
+
+- k8s cluster should be in healthy state including all the nodes in ready state.
+- lvm-controller and csi node-agent daemonset pods should be in running state.
+
+## Exit-Criteria
+
+- lvm-controller statefulset should be scaled up by one replica.
```

Comment:
it gets the initial replica count and increase it by one if schedule-able node is present. So it is already in HA (say repplica:2) then it will scale it up to 3